docs/05-technical-reference/00-custom-resources/oper-01-istio.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Istio Manager Operator
+---
+
+The `istios.operator.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format the Istio Manager Operator listens for. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```shell
+kubectl get crd istios.operator.kyma-project.io -o yaml
+```
+
+More information on the Module Manager you may find [in here](https://github.com/kyma-project/module-manager/).
+
+## Sample custom resource
+
+This is a sample custom resource (CR) that the Istio Manager Operator listens for to manage Istio installation on the cluster. This example has only the **numTrustedProxies** configuration setting which makes the Istio Manager Operator install and configure Istio appropriately. There must be only a single Istio Manager CR in the cluster.
```

Comment:
@pbochynski yes ofc they are already included in design and documented in API proposal in Kyma Istio repository. What we are adding and documenting here is strictly about what's implemented: `numTrustedProxies`. should we link in this document to API proposal in Kyma Istio repo?

Reply:
It is just a configuration you pass to istio operator resource. I expected it to be introduced in one shot. Do we need one sprint for each config option?

Reply:
<img width="485" alt="image" src="https://user-images.githubusercontent.com/29116492/202188053-19708791-b936-4be7-b3a0-438eb6625320.png">

Reply:
They are already implemented 😛 
It is only about making them available for end-user. @tomekpapiernik I see you are bored - should we expect some PRs also?

Reply:
This was clarified, for the moment we will only expose `numTrustedProxies` as configuration since this is the only field that we merge from the CR to the Istio configuration while installing/updating.